web-client/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="uk">
```

Comment:
I don't think uk is a valid iso lang code? https://www.w3schools.com/tags/ref_language_codes.asp

Reply:
Yes, I made a mistake when I tried to fix the coding of the text and wanted to write UA, but wrote UK) - fixed